src/chi/components/header/header.scss

```diff
@@ -443,7 +443,7 @@
         width: 9.0625rem;
 
         @include respond-to(md) {
-          margin: 0 3rem 0 0;
+          margin: 0 4.5rem 0 0;
```

Comment:
There are many scenarios in production (including Control Center) where the header title is rendered within a flat button to trigger a dropdown. In this scenario the title renders 72px from the logo due to the brand margin + the extra padding on the button. If we increase the margin on chi-brand to 4.5rem, it's going to increase the spacing in those scenarios to 96px. I would suggest applying the additional space to chi-header__title, not chi-brand.

Reply:
In the case of the Portal theme Header documentation, there is a custom padding class `-px--1` on the flat dropdown button containing the title which is influencing the spacing between chi-brand and the header title. Removing the class or changing to `-px--2` should get you to 72px. I only see this `-px--1` class in the documentation example. In production we're using `-px--2` and it's rendering correctly at 72px.